### ISSUE TITLE
Future Warning: Using a non-tuple sequence for multidimensional indexing is deprecated;

### ISSUE THREAD

**Issue by rkjulian** [pgmpy/pgmpy#1074]
### Subject of the issue
Running examples throws a "Future Warning" in DiscreteFactor.py line 586

### Your environment
* pypgm 0.1.7
* python 3.7
* OS X

### The following example generates the warning in the program output:

```
from pgmpy.models import BayesianModel
from pgmpy.inference import VariableElimination
from pgmpy.factors.discrete import TabularCPD

restaurant = BayesianModel(
    [('location', 'cost'),
     ('quality', 'cost'),
     ('cost', 'no_of_people'),
     ('location', 'no_of_people')]
)

cpd_location = TabularCPD('location', 2, [[0.6, 0.4]])
cpd_quality = TabularCPD('quality', 3, [[0.3, 0.5, 0.2]])
cpd_cost = TabularCPD('cost', 2,
                      [[0.8, 0.6, 0.1, 0.6, 0.6, 0.05],
                       [0.2, 0.4, 0.9, 0.4, 0.4, 0.95]],
                      evidence=['location', 'quality'], evidence_card=[2, 3])

cpd_no_of_people = TabularCPD('no_of_people', 2,
                              [[0.6, 0.8, 0.1, 0.6],
                               [0.4, 0.2, 0.9, 0.4]],
                              evidence=['cost', 'location'], evidence_card=[2, 2])

restaurant.add_cpds(cpd_location, cpd_quality, cpd_cost, cpd_no_of_people)

restaurant_inference = VariableElimination(restaurant)

loc_pd = restaurant_inference.query(variables=['cost'])

x = loc_pd['cost']

print(x.values)

print(loc_pd['cost'])
```

FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.
 `phi.values = phi.values[slice_]`

### I was able to eliminate the warning by changing line 586:

`phi.values = phi.values[slice_]`

changed to:

`phi.values = phi.values[tuple(slice_)]`

**Comment by ankurankan** [comment]
@rkjulian Thanks for pointing it out. If you would like, you can open a pull request for the changes. Also, we have the same warning at a few other places (ref: https://travis-ci.org/ankurankan/pgmpy/jobs/500727849).

**Comment by rkjulian** [comment]
@ankurankan Thanks. I will get up to speed on the tests and then open a pull request.

**Comment by ankurankan** [comment]
Fixed in #1076
